routes/web.php

```diff
@@ -18,6 +18,8 @@
     return redirect(route('api.settings'));
 });
 
+Route::inertia('/test-inertia', 'SamplePage', [ 'user' => [ 'name' => 'Potato']]);
```

Comment:
In the same vain as the comments above, perhaps we should already scaffold the vue app we want to start soon, so we should probably put this in the "/" route (and move the other one to a different route)

Reply:
Also: :potato: :)

Reply:
Will not address this topic in this PR, we will create a separate task for reorganizing the routes as we agreed.

Reply:
I understand your hesitation to even touch the existing web routes as part of this PR, but I do think that a basic re-structuring just to outline the general structure would actually be very helpful as part of the vue client side setup. Please find a very simple proposal in #63, which will immediately serve the new client side app from "/", without having to type the test URL in the browser's addres bar.

Reply:
ok, voted down